### PR TITLE
chore(doc): Add comment in handleAllEvents

### PR DESCRIPTION
**Pull request by lvalerom**
### Description

Document the `handleAllEvents` function.

### User-facing documentation

- [x] CHANGELOG update is not needed
- [x] Documentation is not needed

### Testing

- [x] inspected CI results

#### Automated testing

(*must be* at least 1 item and all items *must be* checked)

- [x] contributed **no automated tests**
  

#### How I validated my change

This is just a comment

### CODE REVIEW

**Review by vikin91 — APPROVED** [review]

File: sensor/kubernetes/listener/resource_event_handler.go [hunk 1]
```diff
@@ -56,6 +56,20 @@ func managedFieldsTransformer(obj interface{}) (interface{}, error) {
 	return obj, nil
 }
 
+// handleAllEvents starts the dispatchers for all the kubernetes resources tracked by Sensor.
+// For each dispatcher, we wait until it is fully synced, meaning we have received and processed the initial resources
+// from the cluster. This is a synchronous process and can be time costly. It is encouraged to keep the resource
```
Comment (rukletsov) [inline sensor/kubernetes/listener/resource_event_handler.go:61]:
```suggestion
// from the cluster. This is a synchronous process and can be time consuming. It is encouraged to keep the resource
```

**Review by rukletsov — COMMENTED** [review]
Thank Luis, appreciate you taking the time to persist your findings for posterity. 

Visual research suggests the formatting and text width is important for perception. While we don't have strict rules for how comments should be arranged in the code, this is how I would format this blob. Note ~80 char line length, empty lines between paragraphs, shorter sentences. etc

```
/ handleAllEvents starts the dispatchers for all the kubernetes resources
// tracked by Sensor. For each dispatcher, we wait until it is fully synced,
// meaning we received and processed the initial resources from the cluster.
//
// This is a synchronous process and can be time costly. Sensor is an unready
// state until all resource dispatchers finish so keep them quick. Also, the
// go-client documentation recommends swift processing, see
//     https://github.com/kubernetes/client-go/blob/592d891671b2a09e5f81781b28ebe078d8115e41/tools/cache/shared_informer.go#L128-L132
//
// We did some stress testing to determine what would happen if a dispatcher is
// blocked for long periods of time. The results indicated that sensor recovers
// eventually. We should not take this as a valid reason to disregard the
// previous paragraph because the tests were focused on the startup of secrets
// and there are many unknowns regarding other resources. Plus, having this
// function take long is not ideal since other components of Sensor rely on the
// dispatchers to be synced, see e.g.
//     https://github.com/stackrox/stackrox/pull/11662
//
// The order in the startup process is important since some resources depend on
// others. For example, the pod's informer needs to sync before the deployment's
// since the PodLister is used to populate the image ids of deployments.
```

File: sensor/kubernetes/listener/resource_event_handler.go (hunk 1 above)
Comment (lvalerom) [reply sensor/kubernetes/listener/resource_event_handler.go]: Done

**Comment by lvalerom** [comment]
> Visual research suggests the formatting and text width is important for perception. While we don't have strict rules for how comments should be arranged in the code, this is how I would format this blob. Note ~80 char line length, empty lines between paragraphs, shorter sentences. etc

Good to know. Thank you, I've made the changes 👍

**Pull request merged**